pycfmodel/action_expander.py

```diff
@@ -0,0 +1,49 @@
+import re
+from typing import List, Pattern, Union
+
+from pycfmodel.cloudformation_actions import CLOUDFORMATION_ACTIONS
+
+
+def _build_regex(action: str) -> Pattern:
+    # Replace *
+    action = action.replace("*", ".*")
+
+    # Resplace ?
+    action = action.replace("?", ".{1}")
+
+    return re.compile(f"^{action}$", re.IGNORECASE)
+
+
+def _expand_action(action: str) -> List[str]:
```

Comment:
Minor nitpick, I'd change the names of the expand_actions methods to be a bit more different between each other